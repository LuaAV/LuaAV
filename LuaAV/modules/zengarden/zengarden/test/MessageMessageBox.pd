#N canvas 561 22 450 300 10;
#X obj 128 42 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 172 38 loadbang;
#X obj 150 99 f \$0;
#X obj 261 204 r \$0-test;
#X obj 150 248 print direct;
#X obj 261 247 print remote;
#X msg 150 140 \$1-haha \, 2 \, \$1 \; \$1-test 3 \;;
#X connect 0 0 2 0;
#X connect 1 0 2 0;
#X connect 2 0 6 0;
#X connect 3 0 5 0;
#X connect 6 0 4 0;
