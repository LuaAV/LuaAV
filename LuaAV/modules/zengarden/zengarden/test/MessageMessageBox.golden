[@ 0.000ms] direct: 0-haha
[@ 0.000ms] direct: 2
[@ 0.000ms] direct: 0
[@ 0.000ms] remote: 3